src/docsUrl.js

```diff
@@ -1,5 +1,7 @@
+import pkg from '../package'
```

Comment:
it'd be nice to use `.json` here (unless the linter objects)

Reply:
Done.